### ISSUE TITLE
chol() breakage with 0.5

### ISSUE THREAD

**Issue by c42f** [SimonDanisch/FixedSizeArrays.jl#128]
Upstream changes in 0.5 have broken the FixedSizeArray tests again, this time it seems to be changes to the implementation of `chol()`, possibly https://github.com/JuliaLang/julia/pull/16799.

**Comment by c42f** [comment]
This seems to be the cause of errors related to `isreal()` and `ishermitian()` in 0.5 CI at the moment.  I started fixing this, but implementing those predicates wasn't enough, and I need to go on to other things right now.  I'll just quickly paste what I had in here:

``` julia
diff --git a/src/ops.jl b/src/ops.jl
index c93f04f..18ac8df 100644
--- a/src/ops.jl
+++ b/src/ops.jl
@@ -297,6 +297,8 @@ function *(a::FixedMatrix, b::AbstractMatrix)
     Matrix(a)*b
 end

+
+# Predicates
 function (==)(a::FixedVectorNoTuple, b::FixedVectorNoTuple)
     s_a = size(a)
     s_b = size(b)
@@ -322,6 +324,15 @@ end

 (==)(a::AbstractArray, b::FixedArray) = b == a

+isinteger{FSA<:FixedArray}(A::FSA) = all(isinteger, A)
+isinteger{T<:Integer}(A::FixedArray{T}) = true
+
+isreal{FSA<:FixedArray}(A::FSA) = all(isreal, A)
+isreal{T<:Real}(A::FixedArray{T}) = true
+
+ishermitian{FSA<:FixedMatrix}(A::FSA) = (A == A')
+
+
 # To support @test_approx_eq
 Base.Test.approx_full(a::FixedArray) = a
```

**Comment by c42f** [comment]
FIxed in #130 thanks @mschauer !
